modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+      -parameters => {
+        target_db             => $self->o('dna_db'),
+        coord_system_name     => 'toplevel',
+        iid_type              => 'slice',
+        slice_size            => 20000000,
+        include_non_reference => 0,
+        top_level             => 1,
+        min_slice_length      => $self->o('min_toplevel_slice_length'),
+        batch_slice_ids       => 1,
+        batch_target_size     => 20000000,
+      },
+      -rc_name   => '2GB',
+      -flow_into => {
+        '2' => ['remove_redundant_rnaseq_layer_genes_star'],
+      },
+    },
+
+    {
+      -logic_name => 'remove_redundant_rnaseq_layer_genes_star',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::RemoveRedundantGenes',
+      -parameters => {
+        target_db   => $self->o('star_rnaseq_for_layer_nr_db'),
+        target_type => 'generic',
+      },
+      -rc_name => '5GB',
+    },
+
+  ];
+}
+
+sub resource_classes {
+  my $self = shift;
+
+  return {
+    '1GB'     => { LSF => $self->lsf_resource_builder( 'production-rh74', 1000, [ $self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'} ], [ $self->default_options->{'num_tokens'} ] ) },
+    '2GB'     => { LSF => $self->lsf_resource_builder( 'production-rh74', 2000, [ $self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'} ], [ $self->default_options->{'num_tokens'} ] ) },
+    '5GB'     => { LSF => $self->lsf_resource_builder( 'production-rh74', 5000, [ $self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'} ], [ $self->default_options->{'num_tokens'} ] ) },
+    'default' => { LSF => $self->lsf_resource_builder( 'production-rh74', 900,  [ $self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'} ], [ $self->default_options->{'num_tokens'} ] ) },
+    'blast'     => { LSF => $self->lsf_resource_builder( 'production-rh74', 2900,  [ $self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'} ], undef, 3 ) },
+    'blast10GB' => { LSF => $self->lsf_resource_builder( 'production-rh74', 10000, [ $self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'} ], undef, 3 ) },
+    '45GB_star'    => { LSF => $self->lsf_resource_builder( 'production-rh74', 45000, undef, undef, ( $self->default_options->{'star_threads'} + 1 ) ) },
+    '10GB_scallop' => { LSF => $self->lsf_resource_builder( 'production-rh74', 10000, undef, undef, ( $self->default_options->{'scallop_threads'} ) ) },
+    }
+}
+
+sub hive_capacity_classes {
+  my $self = shift;
+
+  return {
+    'hc_very_low' => 35,
+    'hc_low'      => 200,
+    'hc_medium'   => 500,
+    'hc_high'     => 1000,
+  };
+}
+
+sub check_file_in_ensembl {
+  my ( $self, $file_path ) = @_;
+  push @{ $self->{'_ensembl_file_paths'} }, $file_path;
+  return $self->o('enscode_root_dir') . '/' . $file_path;
+}
+
```

Comment:
This can be removed as it is not used anywhere

Reply:
Removed.